integ-test/src/test/java/com/amazon/opendistroforelasticsearch/sql/sql/MathematicalFunctionIT.java

```diff
@@ -0,0 +1,143 @@
+import static com.amazon.opendistroforelasticsearch.sql.util.MatcherUtils.verifySchema;
+import static com.amazon.opendistroforelasticsearch.sql.util.TestUtils.getResponseBody;
+
+import com.amazon.opendistroforelasticsearch.sql.legacy.SQLIntegTestCase;
+import com.amazon.opendistroforelasticsearch.sql.util.TestUtils;
+import java.io.IOException;
+import java.util.Locale;
+import org.elasticsearch.client.Request;
+import org.elasticsearch.client.RequestOptions;
+import org.elasticsearch.client.Response;
+import org.json.JSONObject;
+import org.junit.jupiter.api.Test;
+
+public class MathematicalFunctionIT extends SQLIntegTestCase {
+
+  @Override
+  public void init() throws Exception {
+    super.init();
+    TestUtils.enableNewQueryEngine(client());
+  }
+
+  @Test
+  public void testConv() throws IOException {
+    JSONObject result = executeQuery("select conv(11, 10, 16)");
+    verifySchema(result, schema("conv(11, 10, 16)", null, "string"));
+    verifyDataRows(result, rows("b"));
+
+    result = executeQuery("select conv(11, 16, 10)");
+    verifySchema(result, schema("conv(11, 16, 10)", null, "string"));
+    verifyDataRows(result, rows("17"));
+  }
+
+  @Test
+  public void testCrc32() throws IOException {
+    JSONObject result = executeQuery("select crc32('MySQL')");
+    verifySchema(result, schema("crc32(\"MySQL\")", null, "long"));
+    verifyDataRows(result, rows(3259397556L));
+  }
+
+  @Test
+  public void testMod() throws IOException {
+    JSONObject result = executeQuery("select mod(3, 2)");
+    verifySchema(result, schema("mod(3, 2)", null, "integer"));
+    verifyDataRows(result, rows(1));
+
+    result = executeQuery("select mod(3.1, 2)");
+    verifySchema(result, schema("mod(3.1, 2)", null, "double"));
+    verifyDataRows(result, rows(1.1));
+  }
+
+  @Test
+  public void testRound() throws IOException {
+    JSONObject result = executeQuery("select round(56.78)");
+    verifySchema(result, schema("round(56.78)", null, "double"));
+    verifyDataRows(result, rows(57));
+
+    result = executeQuery("select round(56.78, 1)");
+    verifySchema(result, schema("round(56.78, 1)", null, "double"));
+    verifyDataRows(result, rows(56.8));
+
+    result = executeQuery("select round(56.78, -1)");
+    verifySchema(result, schema("round(56.78, -1)", null, "double"));
+    verifyDataRows(result, rows(60));
+
+    result = executeQuery("select round(-56)");
+    verifySchema(result, schema("round(-56)", null, "long"));
+    verifyDataRows(result, rows(-56));
+
+    result = executeQuery("select round(-56, 1)");
+    verifySchema(result, schema("round(-56, 1)", null, "long"));
+    verifyDataRows(result, rows(-56));
+
+    result = executeQuery("select round(-56, -1)");
+    verifySchema(result, schema("round(-56, -1)", null, "long"));
+    verifyDataRows(result, rows(-60));
+  }
+
+  /**
+   * Test sign function with double value.
+   */
+  @Test
+  public void testSign() throws IOException {
```

Comment:
Is this case and all others in this IT unable to cover in comparison test?

Reply:
the `sign` function in comparison test is covered by H2, but the schema doesn't match for now